addons/mail/static/src/core/common/thread_model.js

```diff
@@ -253,8 +253,11 @@ export class Thread extends Record {
     });
     /** @type {string} */
     defaultDisplayMode;
-    /** @type {SuggestedRecipient[]} */
-    suggestedRecipients = [];
+    suggestedRecipients = Record.attr([], {
+        onUpdate() {
+            this.onUpdateSuggestedRecipients();
```

Comment:
Do you need the method to be defined separately for override? Otherwise it can be put here directly.